system/lib/wasmfs/file.cpp

```diff
@@ -40,44 +40,51 @@ void DataFile::Handle::preloadFromJS(int index) {
 // Directory
 //
 
-void Directory::Handle::cacheChild(const std::string& name,
+void Directory::Handle::cacheChild(const PathString& name,
```

Comment:
Using the `PathString` in each method is not strictly necessary. But I find it easier to enforce which name to use.